src/tests/ftest/cart/test_corpc_exclusive.c

```diff
@@ -126,6 +126,9 @@ int main(void)
 	/* rank, num_attach_retries, is_server, assert_on_error */
 	tc_test_init(my_rank, 20, true, true);
 
+	/* Delay context destruction by 2 seconds after shutdown is issued */
+	tc_set_shutdown_delay(2);
```

Comment:
Must use a macro for time delay

Reply:
For now usage of the function is removed completely, and instead 2 second is added once at init time. Individual tests will still have ability to override this if needed later on

Reply:
But still, wouldn't it be nicer to have a macro name for the default value.  Then it would be much simpler to find it in the individual tests.